### PR TITLE
specs: update web ui spec

### PR DESCRIPTION
**Pull request by yuindustries**

### CODE REVIEW

File: web_ui_awareness.md [hunk 1]
```diff
@@ -1,42 +1,146 @@
 # Web UI Awareness
 
+
+&nbsp;
```
Comment (maiamcc) [inline web_ui_awareness.md]: 🤔

File: web_ui_awareness.md [hunk 2]
```diff
@@ -1,42 +1,146 @@
...
-- When you save a file, Tilt should communicate that it saw the save.
-- You san see that Tilt is reacting to cluster events.
-- You should be able to find the pod ID in the cluster.
-- Tilt should communicate when a service has an endpoint URL that you can open.
-- You should be able to see what Tilt is doing Right Now, with some explanation/justification for why it's doing that.
-- Errors should pop into your view. You shouldn't have to remember to look in an error bucket.
+Changes to the Sidebar, Statusbar, and a new Error pane make it more visible when Tilt responds to file system updates, builds/updates and deploys your resources, and encounters exceptions. 
 
-## Non-Requirements
+Since this is the first pass at a standalone Web UI, we limit our focus to two common use cases:
```
Comment (maiamcc) [inline web_ui_awareness.md]: nice

File: web_ui_awareness.md [hunk 3]
```diff
@@ -1,42 +1,146 @@
...
+&nbsp;
+## Detailed Design
+
+In the default interface, the **overview** panes allow you to navigate to **detail** panes, which either show more information about a single resource (Ex: `Sidebar > doggos > Log`), or show a custom filtered view of your data (Ex: `Sidebar > ALL > Errors`).
+
+***
+The changes support these use cases, which assume you already have a Tiltfile set up:
+
+1. You change code, then see a successful rebuild and deploy
+    - You see Tilt detect the change and start working (and what triggered it)
```
Comment (maiamcc) [inline web_ui_awareness.md]: > (and what triggered it)
wait, i thought we weren't bothering with that right now? (or rather -- this information appears in the logs themselves, but seems incidental / not a thing the web ui is actually doing, and thus maybe not worth including here)

File: web_ui_awareness.md [hunk 4]
```diff
@@ -1,42 +1,146 @@
...
+    - You see Tilt is done (and know your service is ready to check)
+
+2. You introduce errors, see what happened, and how to resolve them.
+    - Build error
+    - Runtime error (after you trigger the error)
```
Comment (maiamcc) [inline web_ui_awareness.md]: you're missing "startup error" (sort of a subcategory of runtime error but we've been discussing them separately)

File: web_ui_awareness.md [hunk 5]
```diff
@@ -1,42 +1,146 @@
...
+&nbsp;
+### STATUSBAR
+1. A single-line message gives you a sense of what Tilt is doing. Messages should be informative, but legible (i.e., if some status would only display for 1ms, show instead the parent process that displays for ~1s)
 
-TODO
+	Example messages:
 
+	```
+	Reading Tiltfile
+	Building: fe
```
Comment (maiamcc) [inline web_ui_awareness.md]: like i mentioned at Planning, imo this shouldn't be "building" but maybe "Updating"? (but also, nbd)

File: web_ui_awareness.md [hunk 6]
```diff
@@ -1,42 +1,146 @@
...
+### ERROR PANE
+`Sidebar > ALL > Errors`
+
+- Even if there are no errors, you can keep this pane open so you'll see errors as soon as they appear.
+- When there are no errors, you see a design that looks intentionally empty.
+
+
+&nbsp;
+### MISC CHANGES
+- See pod ID and and port-forward for each resource
```
Comment (maiamcc) [inline web_ui_awareness.md]: Confused about whether these are goals or non-goals

File: web_ui_awareness.md (hunk 1 above)
Comment (yuindustries) [reply web_ui_awareness.md]: Lol, yes, I am persnickety about layout...

File: web_ui_awareness.md (hunk 5 above)
Comment (yuindustries) [reply web_ui_awareness.md]: A fair point!

File: web_ui_awareness.md (hunk 3 above)
Comment (yuindustries) [reply web_ui_awareness.md]: That phrasing is unclear, so I removed it. Thanks!

File: web_ui_awareness.md (hunk 2 above)
Comment (yuindustries) [reply web_ui_awareness.md]: 😊

**Pull request merged**